module-2/backend/week_two.md

```diff
@@ -5,16 +5,30 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+ - ActiveRecord is Ruby ORM that allows us to interact with/manipulate data in a database as we would a regular Ruby object.
 2. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
+ - `belongs_to` and `has_many` are class variables that create association between other models(classes) in ActiveRecord. EX: A Book(model) belongs_to an Author(model), and a Author has_many Books.
```

Comment:
Think this is kind of a wonky question, but the big idea here is that you can call `book.author`, but not `Book.author` since `.author` is looking for the author of a **particular** book/an **instance** of book.